client/my-sites/post-type-list/post-actions-ellipsis-menu/copy-link.jsx

```diff
@@ -18,7 +18,7 @@ import { infoNotice } from 'state/notices/actions';
 function PostActionsEllipsisMenuCopyLink( { onCopyLinkClick, copyLink } ) {
 	return (
 		<PopoverMenuItemClipboard text={ copyLink } onCopy={ onCopyLinkClick } icon={ 'link' }>
-			{ translate( 'Copy Link' ) }
+			{ translate( 'Copy link' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.